website/src/pages/Home/Home.jsx

```diff
@@ -15,30 +15,37 @@ limitations under the License.
 */
 
 import React, { useContext } from "react";
-import { AppContext } from "../../AppContext";
 
 import logo from "../../assets/logo.png";
 
 import "./home.css";
 
 const Home = () => {
-
   return (
     <div className="home">
-      <article className="home__top justify--content">
-        <div className="home__top__text">
-          <h1 className="header--title">arewefastyet</h1>
-          <span className="header--text">
-            Arewefastyet is all about precise performance measurement.
-            We test Vitess in various scenarios, assessing query latency, transaction speed, and CPU/Memory usage.
-            These insights drive our continuous improvement efforts to ensure Vitess remains at the forefront of performance.
-          </span>
+      <article className="home__top">
+        <div className="home__top__gradient" />
+        <div className="home__top__content">
+          <div className="home__top__content__text">
+            <div className="home__top__content__text__heading">
+              <h3>Vitess Introduces</h3>
+              <h1>arewefastyet</h1>
+            </div>
+            <p>
+            It is all about precise performance measurement. We test Vitess in various scenarios, assessing query latency, transaction speed, and CPU/Memory usage. These insights drive our continuous improvement efforts to ensure Vitess remains at the forefront of performance.
+            </p>
```

Comment:
The `h3` title is always rendered in white, making it invisible when the light-mode is turned on, we should change the color of the title dynamically based on the theme we are using.

Reply:
Moreover - Let's remove the `p` text (`It is all about ...`), and put it in a section underneath the orange block. I feel like this text is too lengthy to be placed at the very top of the page like it is right now. 😃

Reply:
> The h3 title is always rendered in white, making it invisible when the light-mode is turned on, we should change the color of the title dynamically based on the theme we are using.

This problem was caused because of footer, as the specificity of the selectors were not mentioned properly. 
The styles targeted h3, h2, h1 and p directly instead of specifying the .footer part. 
I have fixed it 

Also I noticed in the footer Arewefastyet was written. I understand that it's supposed to be arewefastyet so I changed that as well. Please correct me if I was wrong

Reply:
> Moreover - Let's remove the p text (It is all about ...), and put it in a section underneath the orange block. I feel like this text is too lengthy to be placed at the very top of the page like it is right now.

So shall I put the previous one which was `A Cutting-Edge Benchmarking Approach for Unparalleled Database Speed` here instead of this, or completely get rid of the `p` part

Reply:
> So shall I put the previous one which was A Cutting-Edge Benchmarking Approach for Unparalleled Database Speed here instead of this, or completely get rid of the p part

Yes please :)

Reply:
> Also I noticed in the footer Arewefastyet was written. I understand that it's supposed to be arewefastyet so I changed that as well. Please correct me if I was wrong

That's perfect, thank you!